uahf-technical-spec.md

```diff
@@ -208,6 +219,15 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-6-4 (mandatory use of SCRIPT_VERIFY_STRICTENC)
+
+Once the fork has activated, a transaction shall be deemed invalid if
```

Comment:
This sentence is a bit odd, as far as I know the flag is not shipped *in* the transaction. But this spec gives the impression that that is exactly what is required.

It might be nice to refer to some bip (62 I believe) or other page that explains what this does, as I suspect that btcd etc don't use this exact enum value. And the enum value itself doesn't actually explain enough about the idea to implement it.